manifests/00-cluster-role.yaml

```diff
@@ -182,9 +182,9 @@ rules:
   - '*'
 
 - apiGroups:
-  - maistra.io
+  - sailoperator.io
   resources:
-  - servicemeshcontrolplanes
+  - istios
```

Comment:
And istioRevisions ?

Reply:
Hm, I hadn't thought of that.  However, cluster-ingress-operator doesn't use istiorevisions itself, so it shouldn't need the permission.